class.jetpack-options.php

```diff
@@ -261,7 +327,13 @@ public static function delete_option( $names ) {
 	}
 
 	private static function get_grouped_option( $group, $name, $default ) {
-		$options = get_option( self::$grouped_options[ $group ] );
+		if ( self::is_network_option( $name ) ) {
```

Comment:
I think I don't get the point of your comment here. I'm not messing with grouped options but moving that line.

Instead of ```$options = get_option( self::$grouped_options[ $group ] );```
I use ```$options = get_site_option( self::$grouped_options[ $group ] );```
or the original one ```$options = get_option( self::$grouped_options[ $group ] );``` if it's not a multisite